README.md

```diff
@@ -14,43 +16,134 @@
+
+## Panel works with the tools you know and love
+
+Panel makes it easy to combine widgets, plots, tables and other viewable Python objects into custom analysis tools, applications, and dashboards.
+
+[![Panel Example](https://user-images.githubusercontent.com/42288570/211207118-23d1e6d5-0c47-4489-aab5-892de338cc3b.gif)](https://panel.holoviz.org/reference/templates/FastGridTemplate.html)
+
+<br/>
+
+Panel works really well with Pythons dataviz ecosystem including [Altair](https://altair-viz.github.io/), [Bokeh](https://bokeh.pydata.org), [Folium](https://python-visualization.github.io/folium/), [plotnine](https://plotnine.readthedocs.io/en/stable/), [Matplotlib](https://matplotlib.org/), [Plotly](https://plotly.com/), [pydeck](https://pydeck.gl/), [pyecharts](https://pyecharts.org/#/), [PyVista](https://docs.pyvista.org/), [Seaborn](https://seaborn.pydata.org/) and more. Panel also works with the [ipywidgets](https://ipywidgets.readthedocs.io/en/stable/) ecosystem.
+
+Panel provides advanced bi-directional communication making it possible to react to clicks, selections, hover etc. events.
+
+[![Pythons DataViz works with Panel](https://github.com/MarcSkovMadsen/dataviz-in-python/raw/main/assets/dataviz-in-python.gif)](https://panel.holoviz.org/reference/index.html#panes)
+
+<br/>
+
+Panel is a member of the ambitious [HoloViz](https://holoviz.org/) dataviz ecosystem and has first class support for the other members like [hvPlot](https://hvplot.holoviz.org) (simple .hvplot api),  and [Datashader](https://datashader.org/) (big data viz).
+
+<table style="overflow:hidden">
+  <tr>
+    <td border=1 style="width:12%"><a href="https://panel.holoviz.org"><img style="width:100%;" src="https://holoviz.org/assets/panel.png"></a></td>
+    <td border=1 style="width:10%"><a href="https://hvplot.holoviz.org"><img style="width:100%;" src="https://holoviz.org/assets/hvplot.png"></a></td>
+    <td border=1 style="width:15%"><a href="https://holoviews.org"><img style="width:100%;" src="https://holoviz.org/assets/holoviews.png"></a></td>
+    <td border=1 style="width:14%"><a href="https://geoviews.org"><img style="width:100%;" src="https://holoviz.org/assets/geoviews.png"></a></td>
+    <td border=1 style="width:16%"><a href="https://datashader.org"><img style="width:100%;" src="https://holoviz.org/assets/datashader.png"></a></td>
+    <td border=1 style="width:10%"><a href="https://lumen.holoviz.org"><img style="width:100%;" src="https://holoviz.org/assets/lumen.png"></a></td>
+    <td border=1 style="width:10%"><a href="https://param.holoviz.org"><img style="width:100%;" src="https://holoviz.org/assets/param.png"></a></td>
+    <td border=1 style="width:14%"><a href="https://colorcet.holoviz.org"><img style="width:100%;" src="https://holoviz.org/assets/colorcet.png"></a></td>
+  <tr>
+</table>
```

Comment:
I'm not sure we need this table, it's kind of ugly because the logos don't align well and linking to HoloViz is enough imo.

Reply:
I have tried many solutions without luck. This was the best I could get. It works somewhat ok on desktop. It does not work great on mobile. The problem is that `github´  removes some of the `style` settings.

Do you have a suggestion for something that works.

Reply:
Personally I would just remove it.

Reply:
I've removed it.